JuliaExperimental/gap/singular.g

```diff
@@ -415,14 +415,25 @@ InstallOtherMethod( One, [ "IsSingularObject" ], 200,
 ##  There are high ranked methods for polynomials in GAP,
 ##  thus we have to increase the ranks of our methods.
 ##
+
+
+##############################################################################
+##
+#M  Degree( <pol> ) . . . . . . . . . . . . . .  for a multivariate polynomial
+##
+InstallOtherMethod( Degree,
+    [ "IsSingularPolynomial" ],
+    pol -> Julia.Singular.degree( JuliaPointer( pol ) ) );
```

Comment:
Just in case you are not yet aware of it: If you wrapper objects were in the category `IsJuliaWrapper`, then you could replace `JuliaPointer( pol )` above by just `pol`. And the same in various other places where the wrapper object is passed as argument to a Julia function.

Reply:
Yes, @sebasguts had told me about this feature, and I will use it.

In fact, one can even install `Julia.Singular.degree` itself as a method,
there is no need to wrap it in another GAP function.
(In earlier versions, this was not possible.)